dagshub/data_engine/datasources.py

```diff
@@ -190,6 +190,9 @@ def get_from_mlflow(
     else:
         mlflow_run = run
 
+    if artifact_name is None:
+        raise ValueError("artifact_name must be specified")
```

Comment:
The problem is that making `artifact_name` not optional means that it has to put BEFORE the `run` argument in the function signature. So instead of 
```python
def get_from_mlflow(run, artifact_name)
```

You'll have to make it 
```python
def get_from_mlflow(artifact_name)
```
Which will break the interface in a weirder way than having this ValueError check.